docs/source/guides/gettingstarted.rst

```diff
@@ -71,6 +71,189 @@ An example for this might be:
 
 ``"Path"`` indicates where the script is, ``"RunOn"`` is the Squirrel VM context (see :doc:`../native/sqvm`) as an expression, and ``"ClientCallback"`` and ``"ServerCallback"`` specify a function call that can be ``"Before"`` and/or ``"After"`` map-spawn.
 
+
+Detailed ``mod.json`` architecture
+==================================
+
+Located at your mod's root folder, the ``mod.json`` file is the entrypoint of your mod; 
+it contains human-readable information about it, which scripts to load, and a bunch
+of interesting stuff.
+
+This guide will dig into each of the possible ``mod.json`` fields. Please note that 
+``mod.json`` keys must start with an uppercase letter.
+
+This is what a well-formatted ``mod.json`` looks like:
+
+.. code-block:: json
+
+    {
+        "Name": "Northstar.CustomServers",
+        "Description": "Attempts to recreate the behaviour of vanilla Titanfall 2 servers, as well as changing some scripts to allow better support for mods",
+        "Version": "1.5.0",
+        "LoadPriority": 0,
+        "ConVars": [
```

Comment:
ConVars can have `Flags` too, might be worth adding them here.
Check pins in [#modders](https://discord.com/channels/920776187884732556/922954389415686185) for more information

Reply:
I might also just slap Janu's post into this: https://github.com/R2Northstar/ModdingDocs/issues/19

Reply:
Well, that's what I ended up doing.